flink-python/pyflink/datastream/formats/avro.py

```diff
@@ -63,6 +70,9 @@ class GenericRecordAvroTypeInfo(TypeInformation):
     A :class:`TypeInformation` of Avro's GenericRecord, including the schema. This is a wrapper of
     Java org.apache.flink.formats.avro.typeutils.GenericRecordAvroTypeInfo.
```

Comment:
This class is also a public class which will be used by users and so add it to the __init__.py to expose it?

Reply:
Quite right, I'll update it.